src/reference/asciidoc/aggregator.adoc

```diff
@@ -117,11 +117,15 @@ NOTE: In the interest of code simplicity and promoting best practices such as lo
 Starting with version 5.1, after processing message group, an `AbstractCorrelatingMessageHandler` performs a `MessageBuilder.popSequenceDetails()` message headers modification for the proper splitter-aggregator scenario with several nested levels.
 It is done only if the message group release result is not a message or collection of messages.
 In that case a target `MessageGroupProcessor` is responsible for the `MessageBuilder.popSequenceDetails()` call while building those messages.
```

Comment:
Do you see that this is not true any more?
According your logic no one is responsible any more: the framework does that for us already!

Why don't change this text to reflect reality?

Reply:
If they are using older version and referring to the latest document,  this will reflect that

Reply:
Then they have to refer to documentation for that exact version.
The current doc has to explain exactly the current behavior.

Please, don't confuse people! 😄